app/src/shared/components/DropdownActions/dropdownActions.pcss

```diff
@@ -29,3 +29,7 @@
 .caret {
   margin-left: 0.35em;
 }
+
+.menu {
+  z-index: 10;
```

Comment:
Hm, is this necessary with the z-index fix I proposed?

Reply:
Yes, removed now.